docs/developers/guides/gas/gas-fees.mdx

```diff
@@ -32,75 +42,113 @@ components:
 - **Layer 1 fee** - The L1 fee is the cost to publish your L2 transaction onto Ethereum and can 
     vary based on the blob fee market.
 
-## Estimating transaction costs
+To read more about how gas works on Linea, see [_Gas on Linea_](./gas-on-linea.mdx).
 
-Use the [`linea_estimateGas`](../../reference/api/linea-estimategas.mdx) API method to estimate the 
-gas cost for sending a transaction. The method returns the recommended gas limit, the base fee per 
-gas, and the priority fee per gas.
+## Estimating transaction costs
 
-The `linea_estimateGas` endpoint is currently unavailable. For more information, see 
-our [reference page](../../reference/api/linea-estimategas.mdx).
+:::info
+The best method for estimating gas on Linea, `linea_estimateGas`, is currently unavailable, and will
+be activated soon. See our [reference page](../../reference/api/linea-estimategas.mdx) for more
+information. 
 
-:::info important
+Please get in touch via the [Linea Discord](https://discord.gg/linea) if you intend to use 
+`linea_estimateGas` once it is available.
+:::
 
-Linea also supports [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas),
+Linea supports [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas),
 [`eth_gasPrice`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_gasprice), and
-[`eth_feeHistory`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_feehistory), but 
-they usually return a higher (and less accurate) estimate than `linea_estimateGas`. 
+[`eth_feeHistory`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_feehistory).
 
-We recommend you use `linea_estimateGas` where possible to maximize the accuracy of the gas 
-estimate.
+We suggest you first use `eth_gasPrice` to get the gas price, in wei, and then use this value in 
+`eth_estimateGas`. 
 
-:::
+`eth_estimateGas` returns a total quantity of gas estimated for the transaction, contrasting with
+`linea_estimateGas`, which also will return `baseFeePerGas` and `priorityFeePerGas` once available. 
+`eth_estimateGas` is therefore less precise and generally provides a higher estimate.
 
-The following example executes `linea_estimateGas` on the Linea testnet using the public RPC URL.
+## Step 1: `eth_gasPrice`
```

Comment:
It's probably easier to use eth_feeHistory example. You can pull the `baseFeePerGas` and `reward` from the response